### PR TITLE
JSAPI map examples load cedar and deps via amd

### PR DESCRIPTION
**Pull request by tomwayson**
also updated all urls to cdn deps for docs so they are explicit about versions, etc

### CODE REVIEW

File: site/source/partials/header.hbs [hunk 1]
```diff
@@ -27,14 +27,41 @@
   <!-- stylesheet -->
   <link rel="stylesheet" href="{{assets}}css/style.css">
 
-  <!-- Cedar -->
-
-  <!-- esri Cedar -->
-  <script type="text/javascript" src="http://square.github.io/crossfilter/d3.v3.min.js"></script>
-  <script type="text/javascript" src="{{assets}}js/vega.js"></script>
```
Comment (tomwayson) [inline site/source/partials/header.hbs]: I think we can get rid of the locally hosted copy of vega.js now. LMK if you want me to do that and update the grunt build, etc.

File: site/source/partials/header.hbs [hunk 2]
```diff
@@ -27,14 +27,41 @@
...
+  <!-- Cedar and dependencies (d3, vega) -->
+  <!-- NOTE: the bar chart examle does not work w/ latest vega (v1.5+), so here we are pulling in an oldder version (v1.4.3) -->
```
Comment (tomwayson) [inline site/source/partials/header.hbs]: Damnit, meant to say pie chart example doesn't work w/ latest vega (not bar chart). Prob ought to look into that.

File: site/source/pages/examples/bar-map-integration.hbs [hunk 3]
```diff
@@ -15,10 +16,10 @@ layout: example.hbs
 
 {{#markdown}}
 ```html
-{{>example-code-header}}
+{{>example-code-header-amd}}
```
Comment (tomwayson) [inline site/source/pages/examples/bar-map-integration.hbs]: I think in #96 you get rid of the code example header/footer, but I didn't realize that until after I created the example AMD header. If you want I can get rid of it which might help avert merge conflicts.

File: site/source/partials/header.hbs (hunk 2 above)
Comment (ajturner) [inline site/source/partials/header.hbs]: I'm upgrading to Vega2 - and also PieCharts--. So let's readdress post v2 switch?

File: site/source/partials/header.hbs (hunk 1 above)
Comment (ajturner) [inline site/source/partials/header.hbs]: b/c the vega is included into the Cedar build?

File: site/source/partials/header.hbs (hunk 2 above)
Comment (tomwayson) [inline site/source/partials/header.hbs]: Sounds like a plan.

**Comment by tomwayson** [comment]
Once merged this resolves #63

File: site/source/partials/header.hbs (hunk 1 above)
Comment (tomwayson) [inline site/source/partials/header.hbs]: No. I'm not sure of the history of why the docs site has been pointing to [a locally hosted copy of vega.js](https://github.com/Esri/cedar/blob/master/js/vega.js), but now that I've tested the whole site on CDN hosted versions of vega and we know that the whole site works on v1.5 (except the pie chart examples which require v1.4.3) I think we can get rid of that file and just use the CDN for the site and bower for the builds.

File: site/source/pages/examples/bar-map-integration.hbs (hunk 3 above)
Comment (ajturner) [inline site/source/pages/examples/bar-map-integration.hbs]: yep - get rid of it.

**Pull request merged**